Makefile

```diff
@@ -76,7 +76,7 @@ isort-check:
 
 .PHONY: yamllint
 yamllint:
```

Comment:
Does it have a command to fix in case the linter fails? Maybe add another command here so it fixes all yaml files automatically.

Reply:
Unfortunately [it doesn't support](https://github.com/adrienverge/yamllint/issues/62#issuecomment-322393129) auto fix :/

We could install another tool to do this, but I'm not sure it's worth it. We rarely change the yml files anyway

Reply:
Got it. I agree that it's so rare that it is not worth adding another tool for auto fix.